.gitattributes

```diff
@@ -1,14 +1,12 @@
 * text=auto eol=lf
 
 # Do not include the following files when creating repo artifacts, e.g. the zip
-.editorconfig export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-CHANGELOG.md export-ignore
-CONTRIBUTING.md export-ignore
-LICENSE export-ignore
-phpunit.xml.dist export-ignore
-README.md export-ignore
-tests/ export-ignore
-box.json.dist export-ignore
-build/ export-ignore
+/.* export-ignore
+/build/ export-ignore
+/tests/ export-ignore
+/box.json.dist export-ignore
+/CHANGELOG.md export-ignore
+/CONTRIBUTING.md export-ignore
+/phpunit.xml.dist export-ignore
+/README.md export-ignore
+/demo.gif export-ignore
```

Comment:
`/README.md export-ignore` was set at #1

but, in my opinion, `README.md` should not be export-ignore.

It would be helpful when debugging under installed by composer, to look up `vendor/povils/phpmnd/README.md`.

Other PHP projects also does not do `export-ignore`.

So, I would to remove `/README.md export-ignore` line.

Anyone has other thoughts ?